sphinx/parsers.rst

```diff
@@ -0,0 +1,126 @@
+Parsers
```

Comment:
I'm pretty terrible at `.rst`, but here you go. Based on what you said, this should be suitable for right now.